src/test/java/core/basesyntax/service/impl/StorageServiceImplTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.StorageService;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class StorageServiceImplTest {
```

Comment:
its recommended to have a separate test class for each Class that you want to test, e.g StorageServiceImplTest, ReportCreatorTest ....

Reply:
Ok. Did it